docs/self-managed/zeebe-deployment/configuration/gateway.md

```diff
@@ -170,6 +170,11 @@ security:
 
 ### zeebe.gateway.cluster.security.authentication.identity
 
+:::note
+The Zeebe configuration properties for Camunda Identity are deprecated as of version `8.4.0`. Use the dedicated
+Camunda Identity properties or the [corresponding environment variables](../../identity/deployment/configuration-variables.md#core-configuration).
```

Comment:
Just a thought for maybe @christinaausley - here we link directly to Identity however the configuration, whilst its part of the Identity SDK is actually set within each component. Should we link here instead to the OpenID Connect docs that will be introduced with https://github.com/camunda-cloud/identity/issues/2289#issuecomment-1839081470?

Reply:
Yes -- agreed here @Ben-Sheppard. Could always link to both [Camunda Identity properties] and [corresponding environment variables].